include/common/Elements.hpp

```diff
@@ -21,7 +21,8 @@ namespace arcade::common {
     };
 
     enum class SoundType {
-
+        LOOP,
+        EVENT,
```

Comment:
Should be tempory in game data (remove in next tick)

Reply:
hmmmm that will be a very very fast sound then

Reply:
No need to remove in next tick if we want to use it again, it would be more optimized to store them